tutorial/SparkRedshiftTutorial.scala

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2015 Databricks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.spark.redshift.tutorial
+
+import org.apache.spark.{ SparkConf, SparkContext }
+import org.apache.spark.sql.SaveMode
+import java.io.File
+import org.apache.spark.sql.types.{ StructType, StructField, 
```

Comment:
Please take a look at our code style guide to see how we format / organize imports in Spark, then update this to match.